Makefile

```diff
@@ -17,17 +17,13 @@ find-ignored-tests: ## find all ignored tests (e.g. fdescribe)
 # Documentation targets
 #################################
 
-.PHONY: docs-publish
-docs-publish: ## build and publish docs to github Pages
-	mkdocs gh-deploy
-
```

Comment:
We don't publish the docs of DSP-APP separately any more. So this command isn't used any more.